### PR TITLE
Add ASCII strings proposal writeup

### PR DESCRIPTION
**Pull request by evincarofautumn**
In the future, this kind of thing should probably go into an “RFC” section, but the “advanced” section seems appropriate enough.

### CODE REVIEW

**Comment by akoeplinger** [comment]
A link to this doc from docs/advanced/runtime/docs/index.md would be good (you could add an RFC section there).

**Comment by evincarofautumn** [comment]
Amended to add a link from the index.

**Comment by evincarofautumn** [comment]
Added missing header in `ascii-strings.md` because `htmlproofer` failed.

File: docs/advanced/runtime/docs/ascii-strings.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
+  - /Mono:Runtime:Documentation:AsciiStrings/
+---
```
Comment (akoeplinger) [inline docs/advanced/runtime/docs/ascii-strings.md:1]: This should be:

```
---
title: ASCII Mono
---
```

Comment (akoeplinger) [inline docs/advanced/runtime/docs/ascii-strings.md:1]: for context: Jekyll (the static site generator that processes the website) requires a so called [front matter](https://jekyllrb.com/docs/frontmatter/) block.

**Comment by evincarofautumn** [comment]
Thanks, figured it out just before reading your comment, haha. Checks pass now. :)

**Pull request merged**